package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "lint": "jshint --exclude \"./node_modules/*\" **/*.js",
     "test": "JUNIT_REPORT_PATH=./test/report.xml nyc --reporter=cobertura mocha --colors --reporter mocha-jenkins-reporter",
+    "create-collector": "node create-collector.js",
```

Comment:
All the parameters following after `npm run create-collector` will be passed as is, right?